index.js

```diff
@@ -290,7 +290,7 @@ async function createNewExplanation(info) {
       }}${
 				isInteractiveComicResult
 					? stripIndent`
-          To experience the interactivity, visit the [https://xkcd.com/${comicNum}/ original comic].
+          To experience the interactivity of the game, visit the {{xkcd|{comicNum}|original comic}}!
```

Comment:
In this change, the `comicNum` variable is no longer injected into the string.
```suggestion
          To experience the interactivity of the game, visit the {{xkcd|${comicNum}|original comic}}!
```

Reply:
Yes, thanks! It seems I accidentally removed the $

Reply:
For this, the interactivity is not necessarily a game